src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -81,19 +82,27 @@ private SwitchBootstraps() {}
     private static final MethodHandles.Lookup LOOKUP = MethodHandles.lookup();
     private static final boolean previewEnabled = PreviewFeatures.isEnabled();
 
+    private static final ClassDesc CD_BiPredicate = referenceClassDesc(BiPredicate.class);
```

Comment:
This can use `ReferenceClassDescImpl​::ofValidated​(String)`, which also avoids going through `Class​::descriptorString()` for a statically known class descriptor:
```suggestion
    private static final ClassDesc CD_BiPredicate = ReferenceClassDescImpl.ofValidated("Ljava/util/function/BiPredicate;");
```